src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -1214,5 +1229,61 @@ export var analyticsActionCauseList = {
   foldingShowLess: <IAnalyticsActionCause>{
     name: 'showLessFoldedResults',
     type: 'folding'
+  },
+  /**
+   * Identifies the search event that gets logged when a mLFacet check box is selected and the query is updated.
+   *
+   * `actionCause`: `'facetSelect'`
+   * `actionType`: `'mLFacet'`
+   *
+   * Logging an event with this actionType also adds the following key-value pair in the custom data property of the Usage Analytics HTTP service request.
+   * `"facets":`: <correspondingFacetsState>
+   *
+   * `"facets"` is an Array of objects containing following key-value pairs
+   * `"facetId":`: <correspondingFacetId>
+   * `"facetField":`: <correspondingFacetField>
+   * `"facetTitle":`: <correspondingFacetTitle>
+   * `"facetValue":`: <correspondingFacetValue>
+   * `"facetDisplayValue":`: <correspondingFacetDisplayValue>
+   * `"facetState":`: <correspondingFacetState>
+   */
+  mLFacetSelect: <IAnalyticsActionCause>{
+    name: 'mLFacetSelect',
+    type: 'mLFacet'
+  },
+  /**
+   * Identifies the search event that gets logged when a mLFacet check box is deselected and the query is updated.
+   *
+   * `actionCause`: `'mLFacetDeselect'`
+   * `actionType`: `'mLFacet'`
+   *
+   * Logging an event with this actionType also adds the following key-value pair in the custom data property of the Usage Analytics HTTP service request.
+   * `"facets":`: <correspondingFacetsState>
+   *
+   * `"facets"` is an Array of objects containing following key-value pairs
```

Comment:
If we send all the `facets` state along with the clicked `facetId`, could UA get the info they need? or are the values below not in the state?